WTWSMS/decisions/minor_decisions.txt

```diff
@@ -630,7 +637,7 @@ decisions = {
 				factor = 0.001             # Only a small minority of Christians ever go on a pilgrimage
 			}
 			modifier = {
-				factor = 3
+				factor = 50
```

Comment:
Yes, but this change wasn't needed in the first place. 0.003 is reasonable in my opinion.